### PR TITLE
#1387 fixed classpath regex parsing issue

### LINKED ISSUE #1387: Classpath loading on Windows via gitbash for CobiGen CLI is not working.
### Expected behavior

As a devonfw ide user on Windows, I want to load the CobiGen CLI classes and its dependencies dynamically.
The current classpath in the cobigen start script (executed through git bash) does not include the jars of the "software/cobigen-cli/lib/" folder correctly.

There seems to be an issue with the regex * expression on git bash in Windows.

### Actual behavior

Following error occurs when CobiGen is run:
`Error: Could not find or load main class com.devonfw.cobigen.cli.CobiGenCLI `
`Caused by: java.lang.ClassNotFoundException: com.devonfw.cobigen.cli.CobiGenCLI`

### Steps to reproduce (bug)

1. Install a fresh devonfw ide 2021.04.002 on Windows
2. Open a devon cmd using your context menu inside your project
3. Run: "devon cobigen" to install CobiGen CLI
4. Run "devon cobigen generate"

### Possible fix (Thanks to Jan-Gerrit):
Parsing of the regex expression through a printf
Needs to be adjusted in "software/cobigen-cli/bin/cobigen" file
`cp "$(printf %s: $LIB_DIR/*.jar)"`

### Related Issues:
https://github.com/devonfw/ide/issues/598

### Affected version:

* defonfw ide 2021.04.002
*

### PR DESCRIPTION
**Pull request by jan-vcapgemini**
Fixes issue with classpath parsing on Windows git bash.

Fixes #1387 .

### Related to:
https://github.com/devonfw/ide/issues/598

@devonfw/cobigen

### CODE REVIEW

File: cobigen-cli/cli/src/main/assembly/resources/bin/cg [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 LIB_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../lib" && pwd )"
 if [ -z "$1" ]; then
-    java -cp "$LIB_DIR/*" com.devonfw.cobigen.cli.CobiGenCLI -help
+    java -cp "$(printf %s: $LIB_DIR/*.jar)" -help
```
Comment (SchettlerKoehler) [inline cobigen-cli/cli/src/main/assembly/resources/bin/cg:4]: I think you deleted too much here. "com.devonfw.cobigen.cli.CobiGenCLI" is still needed

**Review by SchettlerKoehler — CHANGES_REQUESTED** [review]

File: cobigen-cli/cli/src/main/assembly/resources/bin/cg (hunk 1 above)
Comment (jan-vcapgemini) [reply cobigen-cli/cli/src/main/assembly/resources/bin/cg]: Oops, fixed now.

File: cobigen-cli/cli/src/main/assembly/resources/bin/cg [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 LIB_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../lib" && pwd )"
 if [ -z "$1" ]; then
```
Comment (SchettlerKoehler) [inline cobigen-cli/cli/src/main/assembly/resources/bin/cg:3]: This looks better, now. Can you please add a comment in the file that -cp "$LIB_DIR/*" is not working? Just to make sure that it is not changed back and everyone knows directly why it is done the way it is done.

**Review by SchettlerKoehler — CHANGES_REQUESTED** [review]

File: cobigen-cli/cli/src/main/assembly/resources/bin/cg (hunk 2 above)
Comment (jan-vcapgemini) [reply cobigen-cli/cli/src/main/assembly/resources/bin/cg:6]: Comment was added.

**Review by SchettlerKoehler — APPROVED** [review]

**Pull request merged**